### PR TITLE
edited test switch X and Y 

### PR DESCRIPTION
**Pull request by PashaBeshchuk**

### CODE REVIEW

File: test/test-tetris.js [hunk 1]
```diff
@@ -191,11 +191,10 @@ describe("I-tetramino rotation 5x5", function() {
 	
 	it('Первый поворот lineTetramino успешный', function(){
 		var lineTetramino = tetris.buildTetramino(`
-			----
-			-X--
-			-X--
-			-X--
-			-X--
+			-----
```
Comment (PashaBeshchuk) [inline test/test-tetris.js]: fix

File: test/test-tetris.js [hunk 2]
```diff
@@ -713,10 +712,10 @@ describe("generation T tetramino", function(){
 		`
 		let transformation = tetris.buildTetramino(stringField)
 		let bodyTetramino =  [
-		   {x: 0, y: 2}, 
+		   {x: 2, y: 0},
```
Comment (PashaBeshchuk) [inline test/test-tetris.js]: all tests names shoud reflect the content

File: test/test-tetris.js [hunk 3]
```diff
@@ -713,10 +712,10 @@ describe("generation T tetramino", function(){
 		`
 		let transformation = tetris.buildTetramino(stringField)
 		let bodyTetramino =  [
-		   {x: 0, y: 2}, 
+		   {x: 2, y: 0}, 
 		   {x: 1, y: 1}, 
-		   {x: 1, y: 2},
-		   {x: 1, y: 3}
```
Comment (PashaBeshchuk) [inline test/test-tetris.js]: Sort X before Y in all tests

Comment (PashaBeshchuk) [reply test/test-tetris.js]: {x: 1, y: 1},
{x: 2, y: 0}
{x: 2, y: 1},
{x: 3, y: 1}

File: test/test-tetris.js [hunk 4]
```diff
@@ -548,7 +548,20 @@ describe("generation tetramino, the size tetramino 5x5", function(){
...
+			--X-
+			--X-
+			--X-
+		`
+		assert.throws(function(){
+			tetris.buildTetramino(stringField)
+		})
+	})
+	
+	/* it("options vertical tetramino x:2 and y:0-4", function(){
```
Comment (zbstof) [inline test/test-tetris.js]: Change into error tests

File: test/test-tetris.js [hunk 5]
```diff
@@ -638,7 +651,7 @@ describe("generation tetramino, the size tetramino 3x3", function(){
 		})
 	})
 	
-	it("options vertical tetramino x:2 and y:0-2", function(){
+	/* it("options vertical tetramino x:2 and y:0-2", function(){
```
Comment (zbstof) [inline test/test-tetris.js]: Change into error tests

File: tetris.js [hunk 6]
```diff
@@ -132,29 +132,49 @@ module.exports = {
...
 					}
+					
 				}
+				elem_Y++
 			}
 			
 		}
-		
-		console.log(arrCoordinats)
+		let condition = true;
```
Comment (zbstof) [inline tetris.js]: maybe describe condition in the name of variable instead of saying just "condition"

File: tetris.js [hunk 7]
```diff
@@ -132,29 +132,49 @@ module.exports = {
 	buildTetramino: function(coordinats){
```
Comment (zbstof) [inline tetris.js]: coordinates

File: tetris.js [hunk 8]
```diff
@@ -132,29 +132,49 @@ module.exports = {
...
 		let math = coordinats.split("\n")
 		let arrCoordinats = []
```
Comment (zbstof) [inline tetris.js]: arrayOfCoordinates

File: tetris.js [hunk 9]
```diff
@@ -132,29 +132,49 @@ module.exports = {
...
+					arrCoordinats[i] = b
+				} 
+				
+			}
+			if(arrCoordinats[0].x <= arrCoordinats[1].x && arrCoordinats[1].x <= arrCoordinats[2].x && arrCoordinats[2].x <= arrCoordinats[3].x){
+				condition = false
+			}					
+		}
+		if(quantityElem != 4){
+			throw error;
```
Comment (zbstof) [inline tetris.js]: variable error is not defined - just throw concrete error

File: test/test-tetris.js [hunk 10]
```diff
@@ -548,84 +548,22 @@ describe("generation tetramino, the size tetramino 5x5", function(){
 		})
 	})
 	
-	it("options vertical tetramino x:2 and y:0-4", function(){
+	it("invalid length tetramino is an error", function(){
```
Comment (zbstof) [inline test/test-tetris.js]: Why do you have horizonatal and vertical error for 3-length tetromino, but only vertical for 5-length tetromino?
Please add test for vertical 5-length tetromino

**Pull request merged**